src/components/SelectionList/SearchTableHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import {View} from 'react-native';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
+
+type SearchTableHeaderProps = {
+    /** Whether we should show the merchant or description column */
+    shouldShowMerchant: boolean;
```

Comment:
For the future cases of mixed reports/expenses- would there be a case where we'd show both the merchant and the description?

Reply:
It's possible, but that'd be a future consideration